derive/src/taptree.rs

```diff
@@ -64,6 +64,14 @@ pub struct TapTreeBuilder {
 impl TapTreeBuilder {
     pub fn new() -> Self { Self::default() }
 
+    pub fn with_capacity(capacity: usize) -> Self {
+        Self {
+            leafs: Vec::with_capacity(capacity),
+            buoy: zero!(),
```

Comment:
What is buoy?

Reply:
A helper structure that assists in constructing and merging merkle trees 

https://github.com/LNP-BP/client_side_validation/blob/08ef1beccac4a833a72264c4047516728f6b2337/commit_verify/src/merkle.rs#L212-L266